packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -20,6 +21,19 @@ class NodeCreate(InputSchemaWithoutCamelCase):
     service_id: str | None
 
 
+class BootOptions(BaseModel):
+    boot_mode: str
+
+
+class NodePatch(InputSchemaWithoutCamelCase):
+    service_version: ServiceVersion = Field(None, alias="version")
+    label: str = Field(None)
+    inputs: InputsDict = Field(None)
+    input_nodes: list[NodeID] = Field(None, alias="inputNodes")
```

Comment:
At the end you do have cammelcase right? Or perhaps ome items in `input_nodes` are not?

Reply:
Yes it should be camelCase